src/domain/authentication/index.types.d.ts

```diff
@@ -0,0 +1,53 @@
+type KratosAuthResponse = {
+  sessionToken: KratosSessionToken
+  kratosUserId: KratosUserId
+}
+
+type LoginWithPhoneNoPasswordSchemaResponse = KratosAuthResponse
+
+type CreateKratosUserForPhoneNoPasswordSchemaResponse = KratosAuthResponse
+
+type KratosUserId = string & { readonly brand: unique symbol }
+type KratosSessionToken = string & { readonly brand: unique symbol }
+
+type KratosError = import("./errors").KratosError
+
+type IdentityKratos = import("@ory/client").Identity
```

Comment:
Should not couple to external dependency in `domain`. The only file / module that should reference `@ory/client` should be `services/kratos/*`.
If we need a type like:
```
type AuthenticatedUserIdentity {
}
```
we should declare it here explicitly and add a translation / adapter in the `kratos` service implementation. This protects `domain` code and anything that depends on it from breaking changes in `@ory/client` (which has happened since we've been using it). Then there is only 1 place in the entire code base (the adapter) that needs changing on breaking updates.